### PR TITLE
Fixed RemoteWindowsDeployer to enable copying  dontet runtime to target server f…

### PR DESCRIPTION
**Pull request by kichalla**
…or enabling portable apps scenario

@JunTaoLuo The changes here are for our CI scenario where we run tests against the latest dotnet runtime. Scenario is (i will send out a PR in music store too) that tests do the setup of dotnet runtime on target server upfront and then run the tests.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/RemotePSSessionHelper.ps1 [hunk 1]
```diff
@@ -10,6 +10,9 @@ param(
 	[string]$accountPassword,
 
 	[Parameter(Mandatory=$true)]
+	[string]$dotnetRuntimePath,
```
Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/RemotePSSessionHelper.ps1]: I don't think this should be a mandatory parameter. We don't need this for standalone apps.

File: src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/RemoteWindowsDeployer.cs [hunk 2]
```diff
@@ -53,6 +53,12 @@ public RemoteWindowsDeployer(RemoteWindowsDeploymentParameters deploymentParamet
                     " Account credentials are required to enable creating a powershell session to the remote server.");
             }
 
+            if (_deploymentParameters.ApplicationType == ApplicationType.Portable
+                && string.IsNullOrWhiteSpace(_deploymentParameters.DotnetRuntimePath))
+            {
+                throw new ArgumentException($"Invalid value '{_deploymentParameters.DotnetRuntimePath}' for {nameof(RemoteWindowsDeploymentParameters.DotnetRuntimePath)}. ");
```
Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/RemoteWindowsDeployer.cs]: Ambiguous error message. Maybe something like "Invalid value for ... DotnetRuntimePath must be non-empty for portable apps."

**Comment by JunTaoLuo** [comment]
:shipit:

**Comment by kichalla** [comment]
bb3555c3dc48cad365277dc7c045be1abf9c5ed6

**Pull request closed without merging**